_includes/head.html

```diff
@@ -7,6 +7,22 @@
 
 {% assign info = site.data.gallery %}
 
+{% elsif page.brand == "illustrations" %}
```

Comment:
vad ca aici folosesti informatiile din _data, pentru a evita if-ul asta imens si sa nu mai faci yml-uiri pentru fiecare pagina, poti pune in fiecare html (vezi review-ul cu *****) texteled in data si sa le folosesti in head.html cu page.canonical de exemplu